scripts/create-torsion-histograms.py

```
@@ -0,0 +1,131 @@
+                mapped = [match[x] for x in map_list]
+                key = f"{mapped[0]}-{mapped[1]}-{mapped[2]}-{mapped[3]}"
+
+                angle = rdMolTransforms.GetDihedralDeg(conf, mapped[0],mapped[1],mapped[2],mapped[3])
+                if (angle < 0.0):
+                    angle += 360.0
+
+                # okay, we want to hash - e.g., 5° bins
+                angle = round(angle / bin_size) % bins
+
+                matched_list[key] = (query, angle)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)